### DISCUSSION TITLE
Organizations on Github

### DISCUSSION THREAD

**Discussion by wumpus27** [community/community#43129] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I‘ll created an free organization on Github and I‘ll would like to name it „OpenWorkspace“, but the name is already used or other. I tried to find this organization/user, but there is an error page ([https://github.com/OpenWorkspace](https://github.com/OpenWorkspace)). I wanted to contact Github to say me, if the organization or user exist yet.

Thank you (I don‘t found the topic organizations in discuss topic select)

**Comment by MitkoVtori** [comment, marked as answer]
### Hi, I see you have created your Organization:

![Screenshot 2023-01-02 162004](https://user-images.githubusercontent.com/112943652/210243863-9f66b12b-10a9-4b37-8fb7-2b88e15b7e43.png)
![Screenshot 2023-01-02 162054](https://user-images.githubusercontent.com/112943652/210243886-31fbe7c2-5e66-41c8-a5f2-f8fcb0122ba2.png)

#### About the link. You've made username, but the original name is OpenWorkspaceHub, so the link will work like this:
(https://github.com/OpenWorkspaceHub).

**Anyways, I don't think there's someone with that name. the only one I found was the same but with an "s" at the end.**

**Reply by wumpus27** [reply]
Thanks, but How I can contact Github (via email or something) to make the name available, when non user or organization is there? I would like to have the name…

**Reply by MitkoVtori** [reply]
> Thanks, but How I can contact Github (via email or something) to make the name available, when non user or organization is there? I would like to have the name…

here's the email: [support@github.com](mailto:support@github.com)
